vespamalloc/src/tests/stacktrace/stacktrace.cpp

```diff
@@ -17,12 +17,22 @@ void * run(void * arg)
 }
 
 void verify_that_vespamalloc_datasegment_size_exists() {
-#pragma GCC diagnostic push
-#if __GNUC_PREREQ(2, 33)
-#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
-#endif
+#if __GLIBC_PREREQ(2, 33)
+    struct mallinfo2 info = mallinfo2();
+    printf("Malloc used %zm of memory\n", info.arena);
```

Comment:
Format string is wrong, missing `d` after `%z`

Reply:
Fixed